multitenancy/tenant-manager-api/src/main/resources/application.properties

```diff
@@ -55,6 +55,7 @@ tenant-manager.date=${timestamp}
 quarkus.micrometer.enabled=true
 quarkus.micrometer.registry-enabled-default=true
 quarkus.micrometer.binder.http-server.enabled=true
+quarkus.micrometer.binder.http-server.ignore-patterns=/q/metrics,/q/health.*,/favicon.ico,NOT_FOUND,root,/
```

Comment:
What's the purpose of this? To not create metrics from the configured paths?

Reply:
yes, to not create metrics for those paths, because those paths cause some bug in micrometer that make it not generate metrics for the paths we are interested in.
for that and just to avoid generating too much metrics